troposphere/elasticsearch.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2013, David Irvine <irvined@gmail.com>
+# All rights reserved.
+#
+# See LICENSE file for full license.
+
+from . import AWSObject, Ref, GetAtt, Tags, AWSProperty
+from .validators import boolean, integer
+
+
+class ElasticSearchEBSOption(AWSProperty):
+    props = {
+        "EBSEnabled": (boolean, False),
+        "Iops": (integer, False),
```

Comment:
The validator for this (and a few others in here) should probably be `positive_integer` I think.